pages/en/about/governance.md

```diff
@@ -22,14 +22,14 @@ For the current list of Collaborators, see the project's [README.md][].
 
 A guide for Collaborators is maintained at [collaborator-guide.md][].
 
-## Technical Steering Committee
+## Top Level Committees
```

Comment:
Why has this been changed?

Reply:
Hmm, a small typo. (I guess this change was recent)

Reply:
```suggestion
## Technical Steering Committee
```